certified-connectors/WorkSpan/apiProperties.json

```diff
@@ -124,58 +125,70 @@
           "x-ms-apimTemplate-operationName": [
             "SubscribeObjectEvent"
-          ],
-          "x-ms-apimTemplateParameter.newPath": "/subscriber"
+          ]
+        }
+      },
+      {
+        "templateId": "routerequesttoendpoint",
+        "title": "Route Submit to Partner Center event subscriber to common subscribe endpoint",
+        "parameters": {
+          "x-ms-apimTemplateParameter.newPath": "/event/v1/subscriber",
+          "x-ms-apimTemplateParameter.httpMethod": "@Request.OriginalHTTPMethod",
+          "x-ms-apimTemplate-operationName": [
+            "SubmitToPartnerCenterObjectEvent"
+          ]
         }
       },
       {
         "templateId": "routerequesttoendpoint",
         "title": "Route bulkload_opportunity request for fileupload",
         "parameters": {
+          "x-ms-apimTemplateParameter.newPath": "/bulk/v1/bulkload_opportunity",
+          "x-ms-apimTemplateParameter.httpMethod": "@Request.OriginalHTTPMethod",
           "x-ms-apimTemplate-operationName": [
             "BulkloadOpportunityFileAttachment"
-          ],
-          "x-ms-apimTemplateParameter.newPath": "/bulkload_opportunity"
+          ]
+        }
+      },
+      {
+        "templateId": "routerequesttoendpoint",
+        "title": "Route bulkload_data in attachment request for fileupload",
+        "parameters": {
+          "x-ms-apimTemplateParameter.newPath": "/bulk/v1/bulkload",
+          "x-ms-apimTemplateParameter.httpMethod": "@Request.OriginalHTTPMethod",
+          "x-ms-apimTemplate-operationName": [
+            "BulkloadDataFileAttachment"
+          ]
         }
       },
       {
         "templateId": "setheader",
         "title": "Set env header to redirect to sandbox host",
         "parameters": {
-          "x-ms-apimTemplate-policySection": "Request",
           "x-ms-apimTemplateParameter.name": "x-ws-env",
           "x-ms-apimTemplateParameter.value": "@connectionParameters('token:HostName','')",
-          "x-ms-apimTemplateParameter.existsAction": "override"
+          "x-ms-apimTemplateParameter.existsAction": "override",
+          "x-ms-apimTemplate-policySection": "Request"
         }
       },
       {
         "templateId": "dynamichosturl",
         "title": "Choose Host for connection",
         "parameters": {
-          "x-ms-apimTemplateParameter.urlTemplate": "@connectionParameters('token:env')/event/v1"
-        }
-      },
-      {
-        "templateId": "dynamichosturl",
-        "title": "Bulk API URL Rewrite",
-        "parameters": {
-          "x-ms-apimTemplate-operationName": [
-            "BulkloadOpportunityFileAttachment",
-            "BulkloadOpportunity"
-          ],
-          "x-ms-apimTemplateParameter.urlTemplate": "@connectionParameters('token:env')/bulk/v1"
+          "x-ms-apimTemplateParameter.urlTemplate": "@connectionParameters('token:env')"
         }
       }
-    ]
+    ],
+    "publisher": "Pooja Dahake"
```

Comment:
@WorkSpan-Engineering The publisher property value should not be an individual name, it can be your company or organization name.

Reply:
We will modify the publisher.